client/src/pages/Result.js

```diff
@@ -1,104 +1,118 @@
 import React, { useState, useEffect } from "react";
-import { BrowserRouter } from "react-router-dom";
-import GlobalStyles from "../GlobalStyles";
+
 import styled from "@emotion/styled";
-import SearchHeader from "../components/SearchHeader";
+import BreadcrumbHeader from "../components/BreadcrumbHeader";
 import Footer from "../components/Footer";
-import List from "../components/List";
-import { fetchResults } from "../api/results";
+import { fetchOffer } from "../api/results";
 import LocationSrc from "../assets/icons/location.svg";
 import DateSrc from "../assets/icons/date.svg";
 import TimeSrc from "../assets/icons/time.svg";
 import fruitsSrc from "../assets/icons/fruits.svg";
+import Button from "../components/Button";
+import Tag from "../components/Tag";
 
 const Main = styled.div`
-  margin: 6em 0;
-  display: flex;
+  margin: 6em 1em;
+  align-items: center;
+  justify-content: center;
```

Comment:
I think `align-items` and `justify-content` has no effect, because it's not a flex box anymore.

Reply:
Absolutely right. Erased now.